docs/components/motor/_index.md

```diff
@@ -18,39 +18,19 @@ hide_children: true
 # SME: Rand
 ---
 
-Electric motors are machines that convert electricity into rotary motion.
-They are the most common form of [actuator](https://en.wikipedia.org/wiki/Actuator) in robotics.
-The _motor_ component type natively supports brushed DC motors, brushless DC motors, and stepper motors controlled by a variety of [motor drivers](https://www.wellpcb.com/what-is-motor-driver.html).
-
-Most machines with a motor need at least the following hardware:
-
-- The motor itself.
-- A compatible motor driver.
-  This takes signals from the computer and sends the corresponding signals and power to the motor.
-  Selected based on the type of motor (for example, brushed, brushless, or stepper) and its power requirements.
-- A [board component](/components/board/) to send signals to the motor driver[^dmcboard].
-  For example, a Raspberry Pi, or another model of single-board computer with GPIO (general purpose input/output) pins.
-
-[^dmcboard]: The `DMC4000` model does not require a board.
-
-{{< cards >}}
-{{< card link="/get-started/control-motor/" class="green">}}
-{{< /cards >}}
-
-## Related services
+Motor components provide an API for operating a motor or getting its current status.
 
-{{< cards >}}
-{{< relatedcard link="/services/frame-system/" >}}
-{{< relatedcard link="/services/motion/" >}}
-{{< relatedcard link="/services/navigation/" >}}
-{{< relatedcard link="/services/slam/" >}}
-{{< /cards >}}
+If you have a physical motor that convert electricity into rotary motion, configure it as a _motor_ component.
 
 ## Available models
 
-To use your motor component, check whether one of the following models supports it.
+To use a motor, you have to add it to your machine's configuration.
+Go to your machine's **CONFIGURE** page, and add a model that supports your motor.
+
+The _motor_ component type natively supports brushed DC motors, brushless DC motors, and stepper motors controlled by a variety of [motor drivers](https://www.wellpcb.com/what-is-motor-driver.html).
```

Comment:
```suggestion
Because your machine controls your motor using a [motor driver](https://www.wellpcb.com/what-is-motor-driver.html), you technically need to select a motor model that supports your motor driver rather than the motor itself.
```